src/vmm/src/vmm_config/drive.rs

```diff
@@ -23,19 +23,14 @@ type Result<T> = result::Result<T, DriveError>;
 /// Errors associated with the operations allowed on a drive.
 #[derive(Debug)]
 pub enum DriveError {
```

Comment:
Could `thiserror::Error` be used on `DriveError` here to avoid needing to write the `Display` implementation manually?

Reply:
Yes,  I created a separate commit for that.